spec/features/posting_message_spec.rb

```diff
@@ -0,0 +1,8 @@
+feature "posts a message" do 
```

Comment:
For the feature testing, one suggestion I have is to create the data you want to test (i.e. Peep.new(...)) and then check to see if the data is present.